pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.baidu.hugegraph</groupId>
     <artifactId>hugegraph-client</artifactId>
-    <version>2.0.1</version>
+    <version>2.1.0</version>
```

Comment:
add java 11 env test?

Reply:
java 11 dependency server, so we can't add java 11 env test later, this commit can  merge first